### DISCUSSION TITLE
Improve docs.github.com accessibility in Dark dimmed mode

### DISCUSSION THREAD

**Discussion by kblok** [community/community#20871] [Accessibility]
The side menus on pages like [this one](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions) are not accessible.  

![image](https://user-images.githubusercontent.com/2198466/179761477-14b1ee87-afff-4344-b9cf-f70454db7d5d.png)

Those links should have an accessible contrast no matter the theme I'm using.

**Comment by cehfisher** [comment]
Hi @kblok - it looks like your issue has been resolved with recent updates for the dark dimmed theme. I'm seeing the background color of rgb(34, 39, 46) and link text color of rgb(83, 155, 245) when I inspect the code, which gives a color contrast ratio of 5.3:1. This color combination passes [WCAG SC 1.4.3 for minimum contrast](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) (level AA).

Please confirm you see these changes and close this issue as resolved. Thanks! (cc @queenofcorgis)
